test/sqllogictest/regex.slt

```diff
@@ -608,6 +608,8 @@ Explained Query:
     Map (regexp_replace["b..", case_insensitive=false, limit=1](#0, "X"))
       ReadStorage materialize.public.r
 
+Source materialize.public.r
+
```

Comment:
This is not due to this PR, but a merge skew on `main`, see [here](https://github.com/MaterializeInc/materialize/pull/27390).

Reply:
I assumed, but wanted a green tick and wasn't brave enough to announce it as skew. :)